core/src/hpke.rs

```diff
@@ -253,13 +253,59 @@ impl HpkeKeypair {
     }
 }
 
+/// HPKE configuration compatible with the output of `divviup hpke-config generate`.
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize)]
+pub struct DivviUpHpkeConfig {
+    id: HpkeConfigId,
+    kem: Kem,
+    kdf: Kdf,
+    aead: Aead,
+    public_key: HpkePublicKey,
+    private_key: HpkePrivateKey,
+}
+
+// We use a fallible TryFrom conversion because it's possible that HpkeDispatch, Janus and
```

Comment:
in this case, if the struct stores `String`s for each of the values, the `Kem/Kdf/Aead as TryFrom<str>` implementations in the latest hpke-config (0.5.1) are far more permissive than serde